README.md

```diff
@@ -210,7 +210,7 @@ After you make changes in `.arb` files you should do additional actions to regen
 * Feature-based folder structure
 * Prefers composition over inheritance
 * Dependency Injection is implemented via `provider`
-* Prefers `StatelessWidget` and async UI update via `StreamBuidler` and `BehaviourSubject` & `StreamController` in controller classes
+* Prefers `StatelessWidget` and async UI update via `StreamBuilder` and `BehaviourSubject` & `StreamController` in controller classes
```

Comment:
Some projects don't like changing changelogs. Happy to drop this or any other change.